website/static/css/site.css

```diff
@@ -1856,6 +1856,14 @@ span#profileFullname{
     margin-bottom: 2px;
     margin-left: 0;
     margin-right: 0;
+    padding-right:3px;
+}
+
+.navbar-form a.btn-sign-up{
```

Comment:
Is there any way we can take of bootstrap's CSS rather than having to add this custom CSS?